spec/support/updatable.rb

```diff
@@ -100,3 +100,28 @@ def self.extract_linked_resource_ids(linked_resource)
     expect(updated_resource.send(test_relation).map(&:id)).to include(*old_ids)
   end
 end
+
+RSpec.shared_examples "supports destroy_links" do
```

Comment:
Can you clean it up then, best not to leave code lying around that should be removed.